agent/exploits/cve_2024_37383.py

```diff
@@ -0,0 +1,36 @@
+"""Agent Asteroid implementation for CVE-2024-37383"""
+
+import re
+
+from agent import definitions
+from agent import exploits_registry
+from agent.exploits import webexploit
+
+VULNERABILITY_TITLE = "XSS via SVG Animate Attributes in Roundcube Webmail"
+VULNERABILITY_REFERENCE = "CVE-2024-37383"
+VULNERABILITY_DESCRIPTION = (
+    "Roundcube Webmail before version 1.5.7 and versions 1.6.x before 1.6.7 contains a "
+    "cross-site scripting vulnerability that can be exploited via SVG animate attributes."
+)
+RISK_RATING = "HIGH"
+
+
+@exploits_registry.register
+class CVE202437383Exploit(webexploit.WebExploit):
+    accept_request = definitions.Request(method="GET", path="/")
+    check_request = definitions.Request(method="GET", path="/")
+    accept_pattern = [re.compile('"rcversion":')]
+
+    # Matches:
+    # - All versions up to 1.5.7 (10507)
```

Comment:
should 1.5.7 be included?

Reply:
From my understanding no, that's why I didn't add it.